storybook/stories/examples/cards.js

```diff
@@ -3,7 +3,7 @@ import React from 'react'
 import { Grid, Layout, Root } from 'reflex'
 import { Box } from '../../shared'
 import styles from './layout.css'
-import { top, horizontalHalf } from '../../shared/marginTypes'
```

Comment:
it's intentional. Things were relying on hardcoded margins and I replaced all these by actual reflex margin properties. So they don't match exactly but I think this is a better use of them.

E.g. in one instance `h1` had a `10px 0` margin so I replaced that with `verticalHalf` which is `12px` instead of `10px`